### PR TITLE
CNFT1-1975 Aria Tab Panel for Patient Profile tabs and h2 added to Entry Modal title that is only used in Patient Profile

### PR DESCRIPTION
**Pull request by mdavis-enquizit**
## Description

Aria Tab Panel for Patient Profile tabs and h2 added to Entry Modal title that is only used in Patient Profile

## Tickets

* [CNFT1-1975](https://cdc-nbs.atlassian.net/browse/CNFT1-1975)

## Checklist before requesting a review
- [ ] PR focuses on a single story
- [ ] Code has been fully tested to meet acceptance criteria
- [ ] PR is reasonably small and reviewable (Generally less than 10 files and 500 changed lines)
- [ ] All new functions/classes/components reasonably small
- [ ] Functions/classes/components focused on one responsibility
- [ ] Code easy to understand and modify (clarity over concise/clever)
- [ ] PRs containing TypeScript follow the [Do's and Don'ts](https://www.typescriptlang.org/docs/handbook/declaration-files/do-s-and-don-ts.html)
- [ ] PR does not contain hardcoded values (Uses constants)
- [ ] All code is covered by unit or feature tests

[CNFT1-1975]: https://cdc-nbs.atlassian.net/browse/CNFT1-1975?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: apps/modernization-ui/src/apps/patient/profile/entry/EntryModal.tsx [hunk 1]
```diff
@@ -27,7 +26,7 @@ export const EntryModal = ({ modal, id, title, children, overflow = false, class
             ref={modal}
             className={classNames(styles.modal, className, { [styles.overflow]: overflow })}>
             <header>
-                <Heading level={2}>{title}</Heading>
+                <h2 className="heading_heading__Xr81l heading_two__eM+ke margin-0">{title}</h2>
```
Comment (adamloup-enquizit) [inline apps/modernization-ui/src/apps/patient/profile/entry/EntryModal.tsx:29]: issue:  The classes `heading_heading__Xr81l` and  `heading_two__eM+ke` are auto generated classes from SASS.  They are not guaranteed to exist.

File: apps/modernization-ui/src/apps/patient/profile/PatientProfile.tsx [hunk 2]
```diff
@@ -137,28 +137,40 @@ export const PatientProfile = () => {
                 <div className="main-body">
                     {patient && summary && <PatientProfileSummary summary={summary} patient={patient} />}
 
-                    <div className="grid-row flex-align-center">
-                        <h6
+                    <div role="tablist" className="grid-row flex-align-center">
+                        <p
+                            role="tab"
```
Comment (adamloup-enquizit) [inline apps/modernization-ui/src/apps/patient/profile/PatientProfile.tsx:142]: suggestion: The `p` element isn't tabable, these should include `tabindex=0` to make them tabable

File: apps/modernization-ui/src/apps/patient/profile/PatientProfile.tsx [hunk 3]
```diff
@@ -137,28 +137,40 @@ export const PatientProfile = () => {
...
+                            aria-labelledby="summary-tabpanel"
```
Comment (adamloup-enquizit) [inline apps/modernization-ui/src/apps/patient/profile/PatientProfile.tsx:143]: question:  isn't this the label for the tab?  not sure if an `aria-labeledby` is needed here

Comment (mdavis-enquizit) [reply apps/modernization-ui/src/apps/patient/profile/PatientProfile.tsx:147]: > question: isn't this the label for the tab? not sure if an `aria-labeledby` is needed here

Button was added so it's possible to change tabs with enter button. Was able to get exact desired design besides blue hover.
Screenshot 2024-02-02 at 9 30 04 AM

Comment (mdavis-enquizit) [reply apps/modernization-ui/src/apps/patient/profile/PatientProfile.tsx:147]: > question: isn't this the label for the tab? not sure if an `aria-labeledby` is needed here

I followed tabpanel https://www.w3.org/WAI/ARIA/apg/patterns/tabs/ 

Screenshot 2024-02-02 at 9 51 11 AM

"Each element with role tabpanel has the property [aria-labelledby](https://w3c.github.io/aria/#aria-labelledby) referring to its associated tab element."

**Review by hclarkEnq — DISMISSED** [review]

**Review by hclarkEnq — APPROVED** [review]

**Review by adamloup-enquizit — APPROVED** [review]

**Pull request merged**